youtube/comments.py

```diff
@@ -121,10 +121,18 @@ def post_process_comments_info(comments_info):
             comment['view_replies_text'] = str(reply_count) + ' replies'
 
 
+        def fmt_num(num):
+            for unit in ['','k']:
+                if num < 1000:
+                    return "%3.1f%s" % (num, unit) if unit else num
+                num /= 1000.0
+            return "%.1f%s" % (num, 'm')
+
+        # if comment['like_count'] = '👍 ' + str(fmt_num(comment['like_count']))
         if comment['like_count'] == 1:
             comment['likes_text'] = '1 like'
         else:
-            comment['likes_text'] = str(comment['like_count']) + ' likes'
+            comment['likes_text'] = str(fmt_num(comment['like_count'])) + ' likes'
```

Comment:
I vote no on this. No need to trim the numbers. I think the like counts are missing commas for thousands but that can be in a separate commit/pull. Best to keep pull requests focused on just the individual change.

Reply:
Why? When you see a Number like 34573215 why would you care about the 215 at the end?
Also "34.6m" takes less space than "34,573,215". It's also what yt does.